src/main/resources/simulator/sim_api.h

```diff
@@ -174,7 +173,7 @@ template <class T> class sim_api_t {
       size_t cmd;
       while(!recv_cmd(cmd));
       switch ((SIM_CMD) cmd) {
-        case RESET: reset(); is_reset = true; is_exit = true; break;
+        case RESET: /** no longer supported! */ break;
```

Comment:
The reason why this is no longer needed is that the new interface treats reset like any other input pin.